### ISSUE TITLE
Finish

### ISSUE THREAD

**Issue by Jiabei1208** [SCCapstone/StockPrediction#122]
The final testwebsite 
http://stockprediction12.pythonanywhere.com

Username: Jiabei
password: 123456He@

**Comment by Jiabei1208** [comment]
![1a199ac20f65925f51574d3c08cbc8e](https://user-images.githubusercontent.com/51316334/126984523-5040a473-d9b8-4d5a-98c2-807e06ae3aa8.png)
homepage

![Uploading 13839cf7f24302cc20bfa518b6cd49d.png…]()
stock compare function 

![2ec0fe48f54012ad483b0f7af228c27](https://user-images.githubusercontent.com/51316334/126984798-837c2202-fc60-4720-b102-ef8ff179b951.png)
salestock buystock and Look up transactions

**Comment by josemvidal** [comment]
Got it.

I only see two commits 6518848264e5a5d623f1b67306e7605c2d54c96f 64f0dc7dfe8b0c414cd286b535d5bf0bcd93dd7a by you, each one just 1 line of code. You have to commit all the code you wrote.

**Comment by Jiabei1208** [comment]
![f96ede8fd3d7196040d0cba0dc0c8e8](https://user-images.githubusercontent.com/51316334/126985110-62d071ce-0283-4990-ab99-d74e1d258a40.png)
prediction

**Comment by Jiabei1208** [comment]
This website is almost rewritten because many the original code cannot be used for my project, so I did not submit the code to github.

**Comment by josemvidal** [comment]
You were expected to expand the team's code, not to write your own.

I setup https://github.com/SCCapstone/StockPrediction2 and gave you write access. Please upload your code there.

**Comment by Jiabei1208** [comment]
ok thanks

**Comment by Jiabei1208** [comment]
Because I couldn't contact my team members and ask them about the meaning of certain codes during coding. So I had to rewrite some key functions.

**Comment by josemvidal** [comment]
Yes, re-writing parts of the existing code is what is expected. Change code, commit changes, repeat.

Starting with a blank slate is not expected.

**Comment by Jiabei1208** [comment]
Many of the original functions cannot be used again, such as stock forecasting, querying the stock market, and displaying the real-time price of the stock market. At first, I thought these were API issues, but I couldn't solve them because I couldn't talk to team members, so I had to rewrite these functions, but these functions are the main functions of the project.

**Comment by josemvidal** [comment]
I setup https://github.com/SCCapstone/StockPrediction2 and gave you write access. Please upload your code there.
